website/docs/requiredprobes.md

```diff
@@ -16,7 +16,17 @@ metadata:
   name: k8srequiredprobes
   annotations:
     metadata.gatekeeper.sh/title: "Required Probes"
-    metadata.gatekeeper.sh/version: 1.0.0
+    metadata.gatekeeper.sh/version: 1.1.0
```

Comment:
Just out of curiosity, how are we deciding on version increments? why `1.1.0` and not `1.0.1`?

Reply:
I was going based off the versioning doc linked in the readme. It says minor updates include parameter changes that are backwards compatible. 

https://docs.google.com/document/u/0/d/1IYiypA-mRcdfSVfmoeyuaeG8XtA1u4GkcqH3kEkv2uw/mobilebasic